src/counter/working_with_contract.md

```diff
@@ -22,6 +22,8 @@ or
 npm run build
 ```
 
+This creates a new [Wasm](https://webassembly.org/) binary file in `./target/res/counter_contract.wasm`. This Wasm file has a [Custom Section](https://webassembly.github.io/spec/core/appendix/custom.html) that explains its interfaces in a way other tooling can use. And the best part: even with this extra information embedded in the Wasm binary, the final result still comes out smaller than contracts built without RAEN, thanks to smart compression. (If you're not used to thinking about the compiled size of server-side code, when working with blockchains, you need to [think about it](https://docs.near.org/docs/concepts/storage-staking).)
```

Comment:
```suggestion
This creates a new [Wasm](https://webassembly.org/) binary file in `./target/res/counter_contract.wasm`. This Wasm file has a [Custom Section](https://webassembly.github.io/spec/core/appendix/custom.html) that explains its interfaces in a way other tooling can use. And the best part: even with this extra information embedded in the Wasm binary, the final result still comes out smaller than contracts built without RAEN, thanks to smart compression. (If you're not used to thinking about the compiled size of server-side code, when working with blockchains, you need to [think about it](https://docs.near.org/docs/concepts/storage-staking) since storage isn't as cheap.)
```

Reply:
Thanks for pointing out the confusing language here; I cleaned it up another way